src/main/java/com/rybka/exception/ZeroCurrencyRateException.java

```diff
@@ -0,0 +1,7 @@
+package com.rybka.exception;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class ZeroCurrencyRateException extends RuntimeException {
```

Comment:
Actually it's not a correct name to represent an issues. 
The problem appears when Currency API doesn't return base currency rate. 
I'd name it as  MissedBaseCurrencyException (you can choose your one)